config/locales/en/course/assessment/answer/answers.yml

```diff
@@ -4,14 +4,25 @@ en:
       answer:
         worksheet:
           comments: 'Comments'
+          reset_answer:
+            button: :'course.assessment.answer.reset_answer.button'
+            warning: :'course.assessment.answer.reset_answer.warning'
         comments_footer:
           comment: 'Comment'
         guided:
           continue: 'Continue'
+          reset_answer:
+            button: :'course.assessment.answer.reset_answer.button'
+            warning: :'course.assessment.answer.reset_answer.warning'
         grading:
           grading: 'Grading'
           maximum_grade: '/ %{maximum_grade}'
           grade: 'Grade: %{grade} / %{maximum_grade}'
         explanation:
           correct: 'Correct!'
           wrong: 'Wrong!'
+        reset_answer:
+          button: 'Reset Answer'
+          warning: >
+            Are you sure you want to do this? This will reset your submission and you will lose
```

Comment:
Would it be better to say "it reverts current answer to template " or sth similar ?   "reset submission" to me is like that I will lose the whole submission.